governance/polls/Add LINK-A to Liquidations 2.0 Framework - April 5, 2021.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Add LINK-A to Liquidations 2.0 Framework - April 5, 2021
+summary: This poll proposes system parameters which could be used to initialize LINK-A with the Liquidations 2.0 Framework.
+discussion_link: https://forum.makerdao.com/t/link-a-liquidations-2-0-parameters/7180
+vote_type: Plurality Voting
+categories:
+   - Collateral
+   - Risk Variable
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Add LINK-A to Liquidations 2.0 Framework - April 5, 2021
+
+The Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the MakerDAO mandated actors. This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for three days beginning on Monday, April 5 at 16:00 UTC.
+
+**This is a binary vote. In a binary vote, you may vote for a single option. You should vote for the option which you prefer. Alternatively, if you would accept either of the two options and have no preference between them, you should vote 'Abstain'.**
+
+## Review
+
+This poll allows the MakerDAO governance community to signal their support or opposition to adding LINK-A to the Liquidations 2.0 Framework in the Maker Protocol with the parameters below:
```

Comment:
Change to:  "This poll allows the MakerDAO Community to signal their support or opposition to adding LINK-A to the Liquidations 2.0 Framework with the parameters below:"